documentation/changelog.rst

```diff
@@ -13,6 +13,7 @@ New features
 * Keyboard control over replay [see `PR #562 <https://www.github.com/FlexMeasures/flexmeasures/pull/562>`_]
 * The ``FLEXMEASURES_MAX_PLANNING_HORIZON`` config setting can also be set as an integer number of planning steps rather than just as a fixed duration, which makes it possible to schedule further ahead in coarser time steps [see `PR #583 <https://www.github.com/FlexMeasures/flexmeasures/pull/583>`_]
 * Different text styles for CLI output for errors, warnings or success messages. [see `PR #609 <https://www.github.com/FlexMeasures/flexmeasures/pull/609>`_]
+* Added API endpoints and webpages `/accounts` and `/account/<id>` to list accounts and show a single account [see `PR #432 <https://github.com/FlexMeasures/flexmeasures/pull/605>`_]
```

Comment:
The PR numbers should generally match, and in this case should both be 605. The corresponding Issue is indeed 432, but in this changelog we always refer to the PR.